src/main/resources/com/google/api/codegen/ruby/version_index.snip

```diff
@@ -36,9 +36,12 @@
 @end
 
 @private readmeModule(indexType, module, iterator, content)
+  @# rubocop:disable LineLength
   @##
```

Comment:
I don't think it's meaningful or conventional, simply aesthetic. FWIW I haven't found any references to it in a style guide.